src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -29,381 +29,380 @@ var SyntheticWheelEvent = require('SyntheticWheelEvent');
+      captured: 'onSeekedCapture',
     },
   },
   seeking: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onSeeking: true}),
-      captured: keyOf({onSeekingCapture: true}),
+      bubbled: 'onSeeking',
+      captured: 'onSeekingCapture',
     },
   },
   stalled: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onStalled: true}),
-      captured: keyOf({onStalledCapture: true}),
+      bubbled: 'onStalled',
+      captured: 'onStalledCapture',
     },
   },
   submit: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onSubmit: true}),
-      captured: keyOf({onSubmitCapture: true}),
+      bubbled: 'onSubmit',
+      captured: 'onSubmitCapture',
     },
   },
   suspend: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onSuspend: true}),
-      captured: keyOf({onSuspendCapture: true}),
+      bubbled: 'onSuspend',
+      captured: 'onSuspendCapture',
     },
   },
   timeUpdate: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onTimeUpdate: true}),
-      captured: keyOf({onTimeUpdateCapture: true}),
+      bubbled: 'onTimeUpdate',
+      captured: 'onTimeUpdateCapture',
     },
   },
   touchCancel: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onTouchCancel: true}),
-      captured: keyOf({onTouchCancelCapture: true}),
+      bubbled: 'onTouchCancel',
+      captured: 'onTouchCancelCapture',
     },
   },
   touchEnd: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onTouchEnd: true}),
-      captured: keyOf({onTouchEndCapture: true}),
+      bubbled: 'onTouchEnd',
+      captured: 'onTouchEndCapture',
     },
   },
   touchMove: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onTouchMove: true}),
-      captured: keyOf({onTouchMoveCapture: true}),
+      bubbled: 'onTouchMove',
+      captured: 'onTouchMoveCapture',
     },
   },
   touchStart: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onTouchStart: true}),
-      captured: keyOf({onTouchStartCapture: true}),
+      bubbled: 'onTouchStart',
+      captured: 'onTouchStartCapture',
     },
   },
   transitionEnd: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onTransitionEnd: true}),
-      captured: keyOf({onTransitionEndCapture: true}),
+      bubbled: 'onTransitionEnd',
+      captured: 'onTransitionEndCapture',
     },
   },
   volumeChange: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onVolumeChange: true}),
-      captured: keyOf({onVolumeChangeCapture: true}),
+      bubbled: 'onVolumeChange',
+      captured: 'onVolumeChangeCapture',
     },
   },
   waiting: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onWaiting: true}),
-      captured: keyOf({onWaitingCapture: true}),
+      bubbled: 'onWaiting',
+      captured: 'onWaitingCapture',
     },
   },
   wheel: {
     phasedRegistrationNames: {
-      bubbled: keyOf({onWheel: true}),
-      captured: keyOf({onWheelCapture: true}),
+      bubbled: 'onWheel',
+      captured: 'onWheelCapture',
```

Comment:
(This might be a bigger rework and probably out of scope of this PR. I haven’t checked if there are any places where we break this convention but I’d be surprised.)